lintreview/review.py

```diff
@@ -196,14 +188,47 @@ def publish_problems(self, problems, head_commit):
         for error in problems:
             error.publish(self._gh, self._number, head_commit)
 
-    def publish_ok_comment(self):
-        if config.get('ADD_OK_LABEL', False):
-            label = config.get('OK_LABEL', IssueLabel.OK_LABEL)
+    def publish_status(self, problem_count):
+        """
+        Update the build status for the tip commit.
+        The build will be a success if there are 0 problems.
+        """
+        state = 'failure'
+        description = 'Lint errors found, see pull request comments.'
+        if problem_count == 0:
+            self.publish_ok_label()
+            self.publish_ok_comment()
+            state = 'success'
+            description = 'No lint errors found.'
+        self._gh.create_status(
+            self._pr.head.sha,
+            state,
+            None,
+            description,
+            'lintreview')
+
+    def remove_ok_label(self):
+        label = config.get('OK_LABEL', False)
+        if label:
+            IssueLabel(label).remove(self._gh, self._number)
+
+    def publish_ok_label(self):
+        """
+        Optionally publish the OK_LABEL if it is enabled.
+        """
+        label = config.get('OK_LABEL', False)
+        if label:
             comment = IssueLabel(label)
```

Comment:
May be this variable should be renamed now?
Not too sure what it should be called though. `label_obj` ?